diff --git a/node_modules/fumadocs-ui/.bun-tag-3288020601b84ac7 b/.bun-tag-3288020601b84ac7
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/components/layout/toc.js b/dist/components/layout/toc.js
index f13f31eb99c360a7b9a804086a7fe49398eb6b94..7b32d8d67a380ba93cfc5dd8eec6b96d4950ab3a 100644
--- a/dist/components/layout/toc.js
+++ b/dist/components/layout/toc.js
@@ -35,7 +35,7 @@ function TOCItem({ item }) {
 }
 const Context = createContext(null);
 export function TocPopover({ open, onOpenChange, ref: _ref, ...props }) {
-    return (_jsx(Collapsible, { open: open, onOpenChange: onOpenChange, ...props, children: _jsx(Context, { value: useMemo(() => ({
+    return (_jsx(Collapsible, { open: open, onOpenChange: onOpenChange, ...props, children: _jsx(Context.Provider, { value: useMemo(() => ({
                 open,
                 setOpen: onOpenChange,
             }), [onOpenChange, open]), children: props.children }) }));
diff --git a/dist/layouts/docs/sidebar.js b/dist/layouts/docs/sidebar.js
index c78a562f41a7d955a37061e7f956f6c0c8138e65..d13991034ea70fc957e9793843dfb6acdc3653e6 100644
--- a/dist/layouts/docs/sidebar.js
+++ b/dist/layouts/docs/sidebar.js
@@ -130,7 +130,7 @@ export function SidebarFolderLink(props) {
 }
 export function SidebarFolderContent(props) {
     const ctx = useInternalContext();
-    return (_jsx(CollapsibleContent, { ...props, className: cn('relative', props.className), children: _jsxs(Context, { value: useMemo(() => ({
+    return (_jsx(CollapsibleContent, { ...props, className: cn('relative', props.className), children: _jsxs(Context.Provider, { value: useMemo(() => ({
                 ...ctx,
                 level: ctx.level + 1,
             }), [ctx]), children: [_jsx("div", { className: "absolute w-px inset-y-0 bg-fd-border start-3" }), props.children] }) }));
